projects_index.yaml

```
@@ -1143,3 +1143,61 @@
   summary:
     en: A solutions for traditional chinese scene text recognition competition
     zh: 繁体中文场景文字辨识竞赛的某个解决方案
+
+- repo_url: https://github.com/csuhan/ReDet
+  paper_url: https://openaccess.thecvf.com/content/CVPR2021/papers/Han_ReDet_A_Rotation-Equivariant_Detector_for_Aerial_Object_Detection_CVPR_2021_paper.pdf
+  type: Official Implementation
+  mmrepos:
+    - MMDetection
+  tags:
+    - Backbone
+    - Detection
+    - Rotated Object Detection
+  summary:
+    en: Official implementation of "ReDet: A Rotation-equivariant Detector for Aerial Object Detection"
+    zn: Official implementation of "ReDet: A Rotation-equivariant Detector for Aerial Object Detection"
```

Please revise the key, use `zh` other than `zn`, and update the Chinese sentence like this:
```suggestion
    zh: "ReDet: A Rotation-equivariant Detector for Aerial Object Detection 官方实现"
```